System/HIDAPI.hsc

```diff
@@ -117,7 +117,7 @@ fromInternalDeviceInfo di = DeviceInfo <$>
   pure (fromIntegral $ _usage di) <*>
   pure (fromIntegral $ _interfaceNumber di)
 
-foreign import ccall unsafe "hidapi/hidapi.h hid_error"
+foreign import ccall safe "hidapi/hidapi.h hid_error"
```

Comment:
This one is not necessary, because it's a plain pointer access:

https://github.com/libusb/hidapi/blob/2551a5d84baca323ca8d61db06464cd1b963c289/linux/hid.c#L1126-L1138

But since it's almost certainly called after an blocking-IO function, I think it makes sense to have it `safe` as well and be on the `safe` side.